external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend

```diff
@@ -26,3 +26,8 @@ SRC_URI[opticalflow.sha256sum] = "e300c02e4900741700b2b857965d2589f803390849e1e2
 
 # No stable URI is available for NVIDIAOpticalFlowSDK
 INSANE_SKIP:append:cuda = " src-uri-bad"
+
+# Build paths are currently embedded
+INSANE_SKIP:${PN}-dev += "buildpaths"
+INSANE_SKIP:${PN}-src += "buildpaths"
+INSANE_SKIP:libopencv-core += "buildpaths"
```

Comment:
Are these caused by our append?

Reply:
No, maybe I should remove that commit

Reply:
Tbh, I was not sure about pushing that 😞

Reply:
Yeah I don't think we need to be in the business of fixing up opencv in this layer

Reply:
Yes, please drop this change.